shared/oae/css/oae.components.css

```diff
@@ -936,6 +936,15 @@ div.oae-thumbnail i.fa {
 
 /* Left-hand navigation collapsed view */
 @media (max-width: 991px) {
+    /* hide from screen readers */
+    .oae-lhnavigation {
+        display: none;
```

Comment:
This changes the animation slightly in that you can see the left hand navigation disappear before the animation is completed. Can we modify this to add a `hide` class when the animation is completed and remove the `hide` class when we start opening the navigation?